chrome/app.js

```diff
@@ -62,8 +62,14 @@ GmailToTrello.App.prototype.bindEvents = function() {
         self.popupView.validateData();
     });
 
+    this.model.event.addListener('onLoadThreadTrelloCardsSuccess', function () {
+        self.popupView.buildPopupHtml(self.model.trello.threadCards);
+    });
+
     this.model.event.addListener('onCardSubmitComplete', function(target, params) {
-        self.popupView.displaySubmitCompleteForm();
+        self.popupView.$gttButton.detach();
+        self.popupView.$popup.detach();
+        self.popupView.buildPopupHtml(self.model.trello.threadCards);
```

Comment:
Rebuild the button and popup after submitting the form, so we can display the green button with the correct text related to the card that was just submitted.